site/en/getstarted/cluster/install_cluster-milvusoperator.md

```diff
@@ -376,8 +374,6 @@ When you no longer need the K8s cluster in the testing environment, you can dele
 
 If you use minikube to install the K8s cluster, run `$ minikube delete`.
```

Comment:
Since this topic only introduces minikube. Can we directly say "When you no longer need the K8s cluster in the testing environment, run`$ minikube delete` to delete it."?

Reply:
Another question: Why do we specifically have to mention "testing environment" here? Is this installation method not recommended for production? Please confirm.

Reply:
[6d34454](https://github.com/milvus-io/milvus-docs/pull/1624/commits/6d344542511247e47f53a3cf620a2f406edebdad)